hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/table/action/commit/SparkInsertOverwritePartitioner.java

```diff
@@ -41,6 +42,25 @@ public SparkInsertOverwritePartitioner(WorkloadProfile profile, HoodieEngineCont
     super(profile, context, table, config);
   }
 
+  @Override
+  public BucketInfo getBucketInfo(int bucketNumber) {
+    BucketInfo bucketInfo = super.getBucketInfo(bucketNumber);
```

Comment:
We decide the operation is `INSERT_OVERWRITE` or `INSERT_OVERWRITE_PARTITION` instead so that the `SparkBucketIndexInsertOverwritePartitioner` is not neede anymore.

Reply:
Sorry, I don't understand what you mean here.
Could you please explain it more?